app/workers/enroll_in_course_worker.rb

```diff
@@ -2,19 +2,19 @@
 require "#{Rails.root}/lib/wiki_course_edits"
 
 class EnrollInCourseWorker
-	include Sidekiq::Worker
+  include Sidekiq::Worker
```

Comment:
There are no significant changes made to this file, only cosmetic ones. I replaced tabs with two spaces